projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+import { SandBox } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { View, Text } from 'react-native';
+/**
+ * Internal dependencies
+ */
+import style from '../../style.scss';
+/**
+ * Types
+ */
+import type { PlayerProps } from './types';
+
+type NativePlayerProps = Pick< PlayerProps, 'html' | 'isRequestingEmbedPreview' | 'isSelected' >;
+
+/**
+ * VideoPlayer react component
+ *
+ * @param {object} props - Component props.
+ * @param {string} props.html - HTML markup for the player.
+ * @param {boolean} props.isRequestingEmbedPreview - Whether the preview is being requested.
+ * @param {boolean} props.isSelected - Whether the block is selected.
+ * @returns {object}                     - React component.
+ */
+export default function Player( {
+	html,
+	isRequestingEmbedPreview,
+	isSelected,
+}: NativePlayerProps ) {
+	// Set up style for when the player is loading.
+	const loadingStyle: { height?: number } = {};
+	if ( ! html || isRequestingEmbedPreview ) {
+		loadingStyle.height = 250;
+	}
+
+	return (
+		<View style={ { ...style[ 'wp-block-jetpack-videopress__video-player' ], ...loadingStyle } }>
+			{ ! isSelected && (
+				<View style={ style[ 'wp-block-jetpack-videopress__video-player-overlay' ] } />
```

Comment:
I noticed this style (`wp-block-jetpack-videopress__video-player-overlay`) is defined in [the stylesheet of the block](https://github.com/Automattic/jetpack/blob/5cc113fe9e9d90e753b9b6098d1ccbff5527f4de/projects/packages/videopress/src/client/block-editor/blocks/video/style.native.scss). On other blocks I see we defined the styles for the specific component ([example](https://github.com/Automattic/jetpack/blob/5cc113fe9e9d90e753b9b6098d1ccbff5527f4de/projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/style.scss)), so I'm wondering if we could do the same here. This way we keep the styles close to the component where it's used.

Reply:
The Web player styles are defined in `editor.scss` ([reference](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/client/block-editor/blocks/video/editor.scss#L14)) . I thought there was some talk of the styles moving away from being tied to the component but I could be wrong.